tests/tpostgres.nim

```diff
@@ -237,6 +237,15 @@ suite "various":
       n.inc
     check n == 2
     db.close()
+    
+  test "enum":
+    type
+      Foo = enum
+        A
+        B
+        C
+    let db = test_open()
+    check Foo(db.getRow(sql"SELECT $1", A).get()[0].i) == A
```

Comment:
`postgres` has `getValue` but it seems to be the same implementation as `dbconnector` (i.e. only works on strings) and so I couldn't use that here.